cpu/stm32/include/can_params.h

```diff
@@ -54,9 +54,16 @@ static const can_conf_t candev_conf[] = {
         .rx_pin = GPIO_PIN(PORT_A, 11),
         .tx_pin = GPIO_PIN(PORT_A, 12),
 #elif defined(CPU_FAM_STM32L4) || defined(CPU_FAM_STM32F4)
+
+#if defined(BOARD_NUCLEO_L432KC)
```

Comment:
ok, then this is not board specific
```suggestion
#if defined(CPU_MODEL_STM32L432KC)
```

Reply:
@donsez ping :wink: